manifests/g/Google/Android/SDKPlatformTools/33.0.3/Google.Android.SDKPlatformTools.locale.en-US.yaml

```
@@ -0,0 +1,36 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS1.CRLF.7-3-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Google.Android.SDKPlatformTools
+PackageVersion: 33.0.3
+PackageLocale: en-US
+Publisher: Google LLC
+PublisherUrl: https://developer.android.com/studio
+PublisherSupportUrl: https://developer.android.com/studio/intro
+PrivacyUrl: https://policies.google.com/privacy
+# Author:
+PackageName: Android SDK Platform-Tools
+PackageUrl: https://developer.android.com/studio/releases/platform-tools
+License: Android Software Development Kit License Agreement
+LicenseUrl: https://developer.android.com/studio/terms
+Copyright: Copyright (c) Google LLC.
+CopyrightUrl: https://developer.android.com/studio/terms
+ShortDescription: Android SDK platform-tools includes tools that interface with the Android platform, such as adb, fastboot, and systrace
```

```suggestion
ShortDescription: Android SDK platform-tools includes tools that interface with the Android platform, such as adb and fastboot.
```
>  This package used to contain systrace, but that has been obsoleted in favor of Studio Profiler, gpuinspector.dev, or Perfetto.
https://developer.android.com/studio/releases/platform-tools